김장현_2200004/직사각형길이찾기.py

```diff
@@ -0,0 +1,10 @@
+t = int(input())
+
+for i in range(1, t+1):
+    a = list(map(int, input().split()))
+    for side in a:
+        if a.count(side) in [1]:
+            print(f'#{i}', side)
+        if a.count(side) in [3]:
+            print(f'#{i}', side) 
```

Comment:
```suggestion
        if a.count(side) in [1, 3]:
            print(f'#{i}', side)
```
`in` 문법으로 `if` 문을 위와 같이 구성하는 방법도 있을 것 같네요 😄